addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
         </config-description>
     </thing-type>
 
+    <bridge-type id="prgbridge">
+        <label>Lutron GRX-PRG or GRX-CI-PRG Bridge</label>
+        <description>Ethernet access point to Lutron Grafik Eye 3x/4x Systems</description>
+
+        <channels>
+            <channel id="buttonpress" typeId="buttonpress"/>
```

Comment:
I'm a little unclear on what this channel does.  Is it mostly for troubleshooting purposes?